src/platform/site-wide/header/helpers/index.js

```diff
@@ -46,105 +42,6 @@ export const toggleMinimalHeader = show => {
   defaultHeader.classList.toggle('vads-u-display--none', show);
 };
 
-export const fetchSearchSuggestions = async searchTerm => {
```

Comment:
The `fetchSearchSuggestions`, `onSearch` and `onSuggestionSubmit` were unused, so I removed them.